core/bones/base.py

```diff
@@ -359,6 +362,8 @@ def fromClient(self, skel: 'SkeletonInstance', name: str, data: dict) -> Union[N
                     parsedVal, parseErrors = self.singleValueFromClient(parsedData[language], skel, name, data)
                     res[language] = parsedVal
                     if parseErrors:
+                        for parseError in parseErrors:
+                            parseError.fieldPath.insert(0,language)
```

Comment:
Please follow PEP8-syntax guideline here as well:
```suggestion
                            parseError.fieldPath.insert(0, language)
```